### PR TITLE
chore: Add excellent_migrations

### PR DESCRIPTION
**Pull request by joshlarson**
Screenshot 2023-12-18 at 3 58 55 PM

Screenshot 2023-12-18 at 4 00 05 PM

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205085395491448

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -155,6 +155,10 @@ config :ueberauth, Ueberauth,
     cognito: nil
   ]
 
+# Don't report issues with migrations from before we introduced excellent_migrations
+# config :excellent_migrations, start_after: "20230908133314"
+config :excellent_migrations, start_after: "20220927170631"
```
Comment (joshlarson) [inline config/config.exs:160]: This is intentionally wrong in order to make this fail the build.

File: mix.exs [hunk 2]
```diff
@@ -60,6 +60,7 @@ defmodule Skate.MixProject do
       {:ex_aws, "~> 2.1"},
       {:ex_doc, "~> 0.27", only: :dev, runtime: false},
       {:ex_machina, "~> 2.7.0", only: :test},
+      {:excellent_migrations, "~> 0.1", only: [:dev, :test], runtime: false},
```
Comment (firestack) [inline mix.exs:63]: question(non-blocking): why do we need this in `:test`?

**Review by firestack — APPROVED** [review]
🎉

**Review by lemald — APPROVED** [review]

File: mix.exs (hunk 2 above)
Comment (joshlarson) [reply mix.exs:63]: Huh... I just grabbed that line from the `excellent_migrations` docs. Let's see if CI likes it if we move it to only be `:dev`.

Comment (joshlarson) [reply mix.exs]: CI liked it!

**Pull request merged**